client/src/elements/molecules/item-list-entry.html

```diff
@@ -2,21 +2,27 @@
   <a route-href="route: item; params.bind: {id: item.id}">
     <require from="./item-list-entry.css"></require>
     <require from="../molecules/delete-button.html"></require>
-    <box-icon if.bind="iconSvg" code.bind="iconSvg" size="medium" class="tool-icon"></box-icon>
-    
+    <span if.bind="item.conf.active" class="q-text-small item-list-entry__state-tag item-list-entry__state-tag--active">${'item.activ' & t}</span>
+    <span if.bind="!item.conf.active" class="q-text-small item-list-entry__state-tag item-list-entry__state-tag--inactive">${'item.inactiv' & t}</span>
+    <box-icon if.bind="iconSvg" code.bind="iconSvg" size="big" class="tool-icon"></box-icon>
+
     <span class="q-text-small">${item.conf.title}</span>
     <span class="q-text-small">${item.conf.createdBy}</span>
     <span class="q-text-small ${item.conf.active ? '' : 'q-text--light'}" if.bind="item.conf.updatedDate">
       ${'general.edited' & t} ${item.conf.updatedDate & timeAgo:7*24*60*60}, ${'general.by' & t} ${item.conf.updatedBy}
     </span>
     <span class="q-text-small ${item.conf.active ? '' : 'q-text--light'}" if.bind="!item.conf.updatedDate">
-        ${'general.edited' & t} ${item.conf.createdDate & timeAgo:7*24*60*60}, ${'general.by' & t}  ${item.conf.createdBy}
+      ${'general.edited' & t} ${item.conf.createdDate & timeAgo:7*24*60*60}, ${'general.by' & t} ${item.conf.createdBy}
     </span>
   </a>
-  <delete-button
-    if.bind="user.roles.includes('poweruser') || user.username === item.conf.createdBy"
-    size="small"
-    action.call="deleteItem()"
-    button-type="secondary">
-  </delete-button>
+
+  <div class="item-list-entry__options-dropdown">
```

Comment:
would be nice to separate this into it's own CustomElement. You could work with `slots` for the entries to be able to do the `if.bind` stuff here.

Reply:
Created a new custom element `list-item-entry-dropdown` -> https://github.com/nzzdev/Q-editor/pull/133/commits/1de499782ea36ac04f0e8474e9bad41ef20ea5fd

Reply:
@manuelroth maybe this can even be named just `dropdown-menu` if it can be used generically.

Reply:
Ah I just now understand what you mean with using <slot>. Rename to dropdown-menu and uses <slot> for the actions -> https://github.com/nzzdev/Q-editor/pull/133/commits/4737c150613546e31665fb9404916dbd7f2d418d